### PR TITLE
Add pdf printer

### PR DESCRIPTION
**Pull request by Davidmattei**

### CODE REVIEW

File: Service/Pdf/DomPdfPrinter.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
```
Comment (dameert) [inline Service/Pdf/DomPdfPrinter.php]: Good idea

File: Service/Pdf/PdfPrintOptions.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        $this->setCompress($options[self::COMPRESS] ?? true);
+        $this->setHtml5Parsing($options[self::HTML5_PARSING] ?? true);
+    }
+
+    public function isCompress(): bool
+    {
+        return $this->compress;
+    }
+
+    public function setCompress(bool $compress): void
```
Comment (dameert) [inline Service/Pdf/PdfPrintOptions.php]: private setters

File: Service/Pdf/PdfPrintOptions.php [hunk 3]
```diff
@@ -6,16 +6,16 @@
 
 class PdfPrintOptions
```
Comment (dameert) [inline Service/Pdf/PdfPrintOptions.php]: make all classes final ?

Comment (Davidmattei) [reply Service/Pdf/PdfPrintOptions.php]: done

**Pull request merged**